lib/zinnia.js

```diff
@@ -196,28 +200,29 @@ export async function run ({
       try {
         await Promise.race(childProcesses)
       } catch (err) {
-        if (shouldRestart) return
-        const errorMsg = err instanceof Error ? err.message : '' + err
-        const message = `Cannot start Zinnia: ${errorMsg}`
-        onActivity({ type: 'error', message })
-        maybeReportCrashToSentry(err)
-        throw err
+        if (!shouldRestart) {
+          const errorMsg = err instanceof Error ? err.message : '' + err
+          const message = `Cannot start Zinnia: ${errorMsg}`
```

Comment:
```suggestion
          const message = `Cannot start ${capitalize(module)}: ${errorMsg}`
```

Reply:
there's no `module` available here unfortunately, since this is past the modules loop. We only have an array of child processes, and in the `catch` handler can't tell which exited (afaik)

Reply:
happy to follow up :)

Reply:
I see. Let's see if this ever becomes a problem in the real world.